CHANGELOG.md

```diff
@@ -7,9 +7,13 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 13.8.1
```

Comment:
Shouldn' this be 'unreleased'? (sorry, I'm a bit late to this one)

Reply:
I messed it up a bit - I meant to release it but forgot to bump the gem files, have done that separately now.